input/fsh/actors.fsh

```diff
@@ -5,7 +5,7 @@ Usage: #definition
 * name = "AUeRequestingPlacer"
 * title = "AU eRequesting Placer"
 * status = #draft
-* description = "The AU eRequesting Placer is a system that is responsible for initiating and creating diagnostic service requests. This system connects to enable requesters to provide the necessary information to create a request, ensuring that required data is accurately collected and provided to service providers with sufficient information for them to fulfil the request."
+* description = "The AU eRequesting Placer is a system that is responsible for creating diagnostic requests. This system connects to enable requesters to provide the necessary information to create a request, ensuring that required data is accurately collected and provided to service providers with sufficient information for them to fulfil the request."
```

Comment:
I don't think this implements the [FHIR-46799](https://jira.hl7.org/browse/FHIR-46799) resolution to replace the description with the text in the Resolution Description.
Please check with @michael-wilson-au.

Reply:
[FHIR-46799](https://jira.hl7.org/browse/FHIR-46799) moved to PR covering consolidation of actor definitions.